app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/widgets/SymbolRepresentation.java

```diff
@@ -783,7 +786,13 @@ public void setSize ( double w, double h ) {
                     {
                         // Open the image from the stream created from the
                         // resource file.
-                        return new Image(ModelResourceUtil.openResourceStream(imageFileName));
+                        InputStream inputStream = ModelResourceUtil.openResourceStream(imageFileName);
+                        if(imageFileName.toLowerCase().endsWith("svg")){
+                            return SVGHelper.loadSVG(inputStream);
+                        }
+                        else{
+                            return new Image(inputStream);
+                        }
                     } catch ( Exception ex ) {
                         logger.log(Level.WARNING, "Failure loading image: ({0}) {1} [{2}].", new Object[] { fileName, imageFileName, ex.getMessage() });
```

Comment:
Similar: ex.getMessage might be empty, and stack trace is lost.
Use `logger.log(Level.WARNING, "....", ex)` to log complete exception with message, trace, cause.